db/migrate/20150928151619_add_role_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddRoleToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :role, :string, null: false, default: "member"
```

Comment:
This is fine in the sense that a user can now have many roles such as "member" or "admin", but for strict admin functionality I usually just add a boolean field called `admin` to the table. Rails will automatically generate a method called `admin?` because it is a boolean field.